onnxruntime/core/graph/contrib_ops/internal_nhwc_onnx_opset.cc

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "internal_nhwc_onnx_opset.h"
+
+#include "onnx/defs/operator_sets.h"
+
+#include "core/graph/constants.h"
+#include "core/graph/contrib_ops/nhwc_inference_context.h"
+
+namespace onnxruntime {
+namespace internal_nhwc_onnx {
+using contrib::NhwcInferenceContext;
+
+template <typename F>
+void RegisterNHWCSchema(F&& f, ::ONNX_NAMESPACE::OpSchema&& schema) {
+  // Need to copy the inferencing function from the temporary OpSchema object
+  auto onnx_inferencing_func = schema.GetTypeAndShapeInferenceFunction();
+  f(std::move(::ONNX_NAMESPACE::OpSchema(schema)
```

Comment:
>move [](http://example.com/codeflow?start=9&length=4)

nit: `std::move` shouldn't be needed since this is already an rvalue

Reply:
```suggestion
  f(::ONNX_NAMESPACE::OpSchema(std::move(schema))
```

Reply:
Compiler says 'no' unfortunately. If I change it as suggested I get this error:

```
5>internal_nhwc_onnx_opset.cc
5>D:\src\github\ort\onnxruntime\core\graph\contrib_ops\internal_nhwc_onnx_opset.cc(37,3): error C2664: '_Ret std::_Func_class<_Ret,onnx::OpSchema &&>::operator ()(onnx::OpSchema &&) const': cannot convert argument 1 from 'onnx::OpSchema' to 'onnx::OpSchema &&'
5>        with
5>        [
5>            _Ret=void
5>        ]
5>D:\src\github\ort\onnxruntime\core\graph\contrib_ops\internal_nhwc_onnx_opset.cc(45,19): message : You cannot bind an lvalue to an rvalue reference
5>C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.32.31326\include\functional(875,10): message : see declaration of 'std::_Func_class<_Ret,onnx::OpSchema &&>::operator ()'
5>        with
5>        [
5>            _Ret=void
5>        ]
```